docs/index.html

```diff
@@ -70,16 +64,81 @@ <h3>Example 3</h3>
   &lt;body&gt;
     &lt;script&gt;
       bootstrapGrist({
-        initialFile: 'investment-research.grist',   // path to .grist
-        name: 'Investment Research',                // optional name
+        initialFile: 'weresheet.grist',   // optional path to .grist
```

Comment:
Why the change in this example?

Reply:
This was moving the weresheet live example from its current location as a bulletpoint.